soulstruct/base/maps/msb/core.py

```diff
@@ -482,6 +484,21 @@ def sort_key(key_value) -> tuple[str, int]:
         module_path = Path(module_path)
 
         game_types_import = f"from soulstruct.{self.GAME.submodule_name}.game_types import *\n"
+
+        if include_vanilla_entities:
+            game_folder = self.GAME.submodule_name
+            # DS1R vanilla_entities are located in PTDE folder
+            if game_folder == 'darksouls1r':
+                game_folder = 'darksouls1ptde'
+            try:
+                append_to_module = PACKAGE_PATH(
+                    f"{game_folder}/events/vanilla_entities/{self.path.stem}_entities.py"
+                ).read_text()
+            except FileNotFoundError:
+                append_to_module = ""
+        else:
+            append_to_module = ""
```

Comment:
This logic was previously only [here](https://github.com/Grimrukh/soulstruct/blob/35ab5065a1466f0ca2791447ebeb36d8408054cf/soulstruct/darksouls1r/project/core.py#L195-L203), so this didn't happen when running the function from [here](https://github.com/Grimrukh/soulstruct/blob/35ab5065a1466f0ca2791447ebeb36d8408054cf/soulstruct/base/project/editors/entities.py#L695).